web/package.json

```diff
@@ -78,10 +79,11 @@
     "eslint-plugin-react-hooks": "4.5.0",
     "eslint-plugin-you-dont-need-lodash-underscore": "^6.12.0",
     "esm": "^3.2.25",
+    "husky": "^8.0.0",
     "lodash.flatmap": "^4.5.0",
     "mini-css-extract-plugin": "1.6.2",
     "ndjson-cli": "^0.3.1",
-    "nodemon": "^1.17.1",
+    "nodemon": "^2.0.16",
```

Comment:
This fixed the annoying error when running `yarn install` and seeing a bunch of errors from `fsevents`/`node-pre-gyp` :)

Reply:
It also fixes a bunch of security vulnerabilities. 👀

I was actually planning on submitting a PR for it after the current PRs that target the lock file was merged.